app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Public Recipes</h1>
+
+<% @public_recipes.each do |recipe| %>
+  <div class="public_recipe_list">
+    <h3><%= link_to recipe.name, recipe_path(recipe, recipe) %></h3>
+    <p>Author: <%= recipe.user.name %></p>
+    <p>Total Food Items: <%= recipe.recipe_foods.count %></p>
+   <p>Total Amount: $<%= recipe.calculate_total_amount %></p>
+  </div>
+  <hr>
+<% end %>
```

Comment:
- [ ] Great job on creating such an engaging and visually appealing UI for your public recipe page! 👏 However, I noticed that your current design deviates from the wireframe provided in the project requirements. To ensure your design aligns with the wireframe and project specifications, I recommend making some adjustments. See below for more details on design differences:

<table>
<tr>
<th align=“middle”>Original design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="1140" src="https://github.com/over-geek/Recipe-app/assets/107192076/61d7b8aa-7fb5-47c7-bdc4-7b68eaced9d1">
</td>
<td>
<img width="1440" alt="Screenshot 2023-10-20 at 22 22 17" src="https://github.com/over-geek/Recipe-app/assets/107192076/d4cf8de3-9c77-4b8f-a425-71e34b0c152d">
</td>
</tr>
</table>

Aligning your design with the wireframe not only ensures consistency but also demonstrates your keen attention to detail. 🧐
Happy coding! 😊👨‍💻🚀